src/lang/wasm.ts

```diff
@@ -279,3 +282,11 @@ export function programMemoryInit(): ProgramMemory {
     throw kclError
   }
 }
+
+export async function copilotLspRun(config: ServerConfig, token: string) {
+  await copilot_lsp_run(config, token)
+}
+
+export async function kclLspRun(config: ServerConfig) {
+  await kcl_lsp_run(config)
+}
```

Comment:
Should this maybe be?
```suggestion
export async function copilotLspRun(config: ServerConfig, token: string) {
  await initPromise
  await copilot_lsp_run(config, token)
}

export async function kclLspRun(config: ServerConfig) {
  await initPromise
  await kcl_lsp_run(config)
}
```

Reply:
The reason why `export const initPromise = initialise()` is done at the module level is so that it kicks off the download asap, 
and the reason it's `export const initPromise = initialise()` and not `export const initPromise = await initialise()` is because we want anything that needs wasm to be downloaded to await the same promise, so we're only doing the download once.

Because promises in JS are eager, there's no harm in having multiple awaits for the same promise, because if it's already resolved there will be no wait anyway.